src/less/components/dialog.less

```diff
@@ -50,6 +50,14 @@
                 &.focus {
                     color: @brand-secondary-dark-2;
                     text-decoration: none;
+                    background-color: @brand-bg-gray;
```

Comment:
Close button on sheet and toast does not have this background-color on focus. Should it be added? Or should this be removed?

Reply:
Ah.. it's focus! Maybe remove it? The border should be enough!